enhancements/cluster-scope-secret-volumes/csi-driver-host-injections.md

```diff
@@ -431,12 +261,27 @@ Some links related to that testing:
 
 ### Graduation Criteria
 
-We will not want to GA this EP as is until the related work noted in the Summary is complete.
+#### Developer Preview
 
+While the CSI driver is in developer preview state, cluster admins should be able to install via YAML manifests defined in GitHub.
+
+#### Tech Preview
```

Comment:
Since we are delivering CSI drivers on a tech preview basis, there is no issue IMO providing this new driver as tech preview in the payload.

Reply:
SGTM